app/models/miq_server/log_management.rb

```diff
@@ -142,12 +146,15 @@ def post_one_log_pattern(pattern, logfile, log_type)
     task.update_status("Active", "Ok", msg)
   end
 
-  def post_automate_models(taskid, log_depot)
+  def post_automate_models(taskid, log_depot, log_file)
     domain_zip = Rails.root.join("log", "domain.zip")
     backup_automate_models(domain_zip)
 
     logfile = LogFile.historical_logfile
-    logfile.update(:file_depot => log_depot, :miq_task => MiqTask.find(taskid))
+    logfile.update(:file_depot         => log_depot,
+                   :miq_task           => MiqTask.find(taskid),
+                   :logging_started_on => log_file.logging_started_on,
```

Comment:
What date is important for models/dialogs?  The date of the logs you're collecting or the current date as of the moment of the collection?  I wonder if we can just set started_on and ended_on (if needed) to Time.now or whatever format is expected.

Reply:
Yeah that would make this heckin easier.